python/datafusion/__init__.py

```diff
@@ -36,6 +38,7 @@
     "ScalarUDF",
     "column",
     "literal",
+    "functions",
```

Comment:
this is the fix

Reply:
Actually this should be done by importing the functions submodule:

```py
import datafusion
import datafusion.functions

datafusion.functions.abs(datafusion.column("a"))
```

Reply:
yeah, importing `datafusion.functions` works as expected currently, this change is to make the code in our python doc work again: https://arrow.apache.org/datafusion/python/index.html#how-to-use-it. @kszucs is there any downside in making `f = datafusion.functions` work on top of `import datafusion.functions as f`?